template/web/src/store/createWagmiConfig.ts

```diff
@@ -0,0 +1,34 @@
+import { connectorsForWallets } from '@rainbow-me/rainbowkit';
+import { metaMaskWallet, rainbowWallet, coinbaseWallet } from '@rainbow-me/rainbowkit/wallets';
+import { createConfig, http } from 'wagmi';
+import { base, baseSepolia } from 'wagmi/chains';
+import { getChainsForEnvironment } from './supportedChains';
+
+export function createWagmiConfig(projectId: string) {
+  const connectors = connectorsForWallets(
+    [
+      {
+        groupName: 'Recommended Wallet',
+        wallets: [coinbaseWallet],
+      },
+      {
+        groupName: 'Other Wallets',
+        wallets: [rainbowWallet, metaMaskWallet],
```

Comment:
nit: we should add more wallets here

Reply:
Will take this up as a separate PR to keep each PR atomic and small